README.md

```diff
@@ -1 +1,52 @@
-# ruby-milestone-setupFirst commit development
+# OOP (Object Oriented Programming)
+# Tic Tac Toe Game
+
+# A project made in Ruby to serve as a Tic Tac Toe Game for educational purposes
+
+![screenshot](./screenshot.png)
+
+
+## Built With
+
+- Ruby
+
+## Getting started
+    In order to run the project locally clone the repository and run the command bellow
+    - ruby bin/main.rb
```

Comment:
Explain clearly step by step how to install this game locally and how to start it, and explicitly show point out commands to be used on each step, for example you didn't explain how a user should clone this repo (All users aren't yet familliar with git/github) you should elaborate clear step by step how they get this game on local computers.